resources/views/individuals/edit/experiences.blade.php

```diff
@@ -52,7 +52,7 @@
             </fieldset>
 
             <fieldset class="stack">
-                <legend>{{ __('Relevant experiences') }} {{ __('(optional)') }}</legend>
+                <legend class="h4">{{ __('Relevant experiences') }} {{ __('(optional)') }}</legend>
```

Comment:
I think there's a thick divider missing between Skills and strengths and relevant experiences.